ui-modules/blueprint-composer/app/components/catalog-saver/catalog-saver.modal.template.html

```diff
@@ -57,19 +60,17 @@ <h3 class="modal-title">{{getTitle()}}</h3>
                 <label class="control-label">Bundle ID</label>
                 <div class="input-group">
                     <span class="input-group-addon">catalog-bom-</span>
-                    <input ng-model="config.bundle" ng-disabled="state.saving" class="form-control" placeholder="E.g my-bundle" name="bundle" required ng-pattern="state.pattern" autofocus />
+                    <input ng-model="config.bundle" ng-disabled="state.saving" class="form-control" name="bundle" ng-pattern="state.pattern" autofocus placeholder="{{ config.name | bundlize }}"/>
```

Comment:
I get that `config.name` is required but until you set it, there will be no placeholder. So you should do `{{(config.name | bundlize) || 'E.g my-bundle'}}`